doc/man/lttng-relayd.8.txt

```diff
@@ -220,6 +220,11 @@ Supported on Linux only.
 +
 Search for `tcp_keepalive_intvl` in man:tcp(7) for more information.
 
+`LTTNG_RELAYD_CWD`::
```

Comment:
Why is this named `CWD` and the equivalent option `--working-directory`? Can we have something consistent?

Also, the environment variables are alphabetically sorted in this section. You need to insert this new block at the right location, whatever the name you choose.

Reply:
After discussing with @eepp, we concluded that LTTNG_RELAYD_WORKING_DIRECTORY is a better alternative to LTTNG_RELAYD_CWD. The CWD acronym is not specific/clear enough.